ci-operator/config/openshift/cluster-api-provider-vsphere/openshift-cluster-api-provider-vsphere-master.yaml

```diff
@@ -54,7 +54,12 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+test_binary_build_commands: openshift/unit-tests.sh
```

Comment:
Why is this required? (And throughout)

Reply:
Lets not add it if we don't think we need it. That'll also reduce the likelihood of us cargo-culting it somewhere else later.

Reply:
It looks like it's creates a new image layer by executing `test_binary_build_commands` and tagging the result as `test-bin`. As unit-tests.sh doesn't produce any binary artifacts and in fact runs the full test suite, I think we just end up running the unit tests an extra time during the build step. I'm pretty sure we don't want that.

Reply:
Ok cool. Updated.